src/chart.zoom.js

```diff
@@ -317,16 +317,14 @@ var zoomPlugin = {
 				var tickOptions = scale.options.ticks;
 
 				if (timeOptions) {
-					delete timeOptions.min;
-					delete timeOptions.max;
+					timeOptions.min = scale.originalOptions.time.min;
+					timeOptions.max = scale.originalOptions.time.max;
 				}
 
 				if (tickOptions) {
-					delete tickOptions.min;
-					delete tickOptions.max;
+					tickOptions.min = scale.originalOptions.ticks.min;
+					tickOptions.max = scale.originalOptions.ticks.max;
 				}
-
-				scale.options = helpers.configMerge(scale.options, scale.originalOptions);
 			});
 
 			helpers.each(chartInstance.data.datasets, function(dataset, id) {
```

Comment:
This is could also be deleted (row 330-331), if you have multiple datasets and you faded them out, a resetZoom fades them in again. It is a nice feature, but not always wanted. The user should be able to choose between fading in and stay hidden.